webapp/src/components/github_issue/index.tsx

```diff
@@ -0,0 +1,118 @@
+    theme: Theme,
+    post: Post,
+}
+
+const GithubIssue = ({theme, post}: GithubIssueProps) => {
+    const style = getStyle(theme);
+    const postProps = post.props || {};
+    let assignees;
+    let labels;
+    const dispatch = useDispatch();
+
+    const issue = {
+        repo_owner: postProps.repo_owner,
+        repo_name: postProps.repo_name,
+        issue_number: postProps.issue_number,
+        postId: post.id,
+        status: postProps.status,
+    };
+
+    const content = (
+        <div>
+            <button
+                style={{...style.button, ...style.other_buttons}}
+                className='btn btn-primary'
+                onClick={() => dispatch(attachCommentIssueModal(issue))}
+            >{'Comment'}</button>
+            <button
+                style={{...style.button, ...style.other_buttons}}
+                className='btn btn-primary'
+                onClick={() => dispatch(editIssueModal(issue))}
+            >{'Edit'}</button>
+            <button
+                style={{...style.button, ...{...postProps.status === 'Close' ? style.close_or_reopen_button : style.other_buttons}}}
+                className='btn btn-primary'
+                onClick={() => dispatch(closeOrReopenIssueModal(issue))}
+            >{postProps.status}</button>
+        </div>
+    );
+
+    if (postProps.assignees?.length) {
+        assignees = (
+            <div style={style.assignees_and_labels}>
+                <b>{'Assignees'}</b>
+                <div>
+                    {postProps.assignees.map((assignee: string, index: number) => (
+                        <span key={assignee}>{(index ? ', ' : '') + assignee} </span>
+                    ))}
+                </div>
+            </div>
+        );
+    }
+
+    if (postProps.labels?.length) {
+        labels = (
+            <div style={style.assignees_and_labels}>
+                <b>{'Labels'}</b>
+                <div>
+                    {postProps.labels.map((label: string, index: number) => (
+                        <span key={label}>{(index ? ', ' : '') + label} </span>
+                    ))}
+                </div>
+            </div>
+        );
+    }
+
+    return (
+        <div>
+            <h5>
+                <a
+                    href={postProps.issue_url}
+                    target='_blank'
+                    rel='noopener noreferrer'
+                >
+                    {'#' + postProps.issue_number + ' ' + postProps.title}
+                </a>
+            </h5>
+            <p>{postProps.description}</p>
+            {assignees}
+            {labels}
+            {content}
+        </div>
+    );
+};
+
+const getStyle = makeStyleFromTheme((theme) => ({
+    button: {
+        fontFamily: 'Open Sans',
+        fontSize: '12px',
+        fontWeight: 'bold',
+        letterSpacing: '1px',
+        lineHeight: '19px',
+        margin: '12px 12px 8px 0px',
+        borderRadius: '4px',
+        color: theme.buttonColor,
+    },
```

Comment:
Instead of these styles, can we make the buttons match the styling of most other buttons in the MM UI? There are class names used for this, in the footer of the modals in this project for instance. The current styling seems inconsistent with the rest of the application

Reply:
@DHaussermann I'm curious about your thoughts on this. Should these buttons be made to look like the slack attachment buttons as much as possible?